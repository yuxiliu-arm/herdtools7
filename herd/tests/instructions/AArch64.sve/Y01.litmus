AArch64 Y01
(* EOR *)
{
  uint64_t t[2] = {1,2};
  uint64_t s[2] = {3,4};
  0:X0=t;
  0:X1=s;
}
P0                    ;
PTRUE P0.D,VL2        ;
LD1D {Z0.D},P0/Z,[X0] ;
LD1D {Z1.D},P0/Z,[X1] ;
EOR Z0.D,Z0.D,Z1.D    ;
ST1D {Z0.D},P0,[X0]   ;
forall (
  t={2,6} /\
  s={3,4}
)
