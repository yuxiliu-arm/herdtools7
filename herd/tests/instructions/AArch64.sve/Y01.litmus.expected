Test Y01 Required
States 1
s={3,4}; t={2,6};
Ok
Witnesses
Positive: 1 Negative: 0
Flag Scalable-Vector-Extensions-is-work-in-progress
Condition forall (t={2,6} /\ s={3,4})
Observation Y01 Always 1 0
Hash=c4df84c412ce399286611bf8e695a12a

